c10/util/Backtrace.cpp

```diff
@@ -115,7 +120,22 @@ c10::optional<FrameInformation> parse_frame_information(
   frame.function_name = demangle(mangled_function_name.c_str());
   return frame;
 }
-
+} // anonymous namespace
+#elif defined(_MSC_VER)
+namespace {
+const int max_name_len = 256;
+void get_module_name(char* module, void* addr) {
```

Comment:
nit: this is kind of frustratingly low level for C++. Wonder if we can just return a `std::string` here.